Makefile

```diff
@@ -12,19 +12,8 @@ dummyusers:
 prodceleryworkers:
 	cd contentcuration/ && celery -A contentcuration worker -l info
 
-setupnanobox:
-	nanobox evar add dry-run DJANGO_SETTINGS_MODULE=contentcuration.dry_run_settings
-	nanobox evar add dry-run DJANGO_SETTINGS_MODULE=contentcuration.dev_settings
-	nanobox evar add local MINIO_ACCESS_KEY=development
-	nanobox evar add local MINIO_SECRET_KEY=development
-	nanobox evar add dry-run MINIO_ACCESS_KEY=development
-	nanobox evar add dry-run MINIO_SECRET_KEY=development
-	nanobox evar add local MINIO_RUN_TYPE=LOCAL
-	nanobox evar add dry-run MINIO_RUN_TYPE=LOCAL
-
-
-devserver:
-	yarn run devserver
```

Comment:
pero like why?

By doing this, we're completely ignoring the scripts I purposely wrote into `package.json` to run minio, Django, and Webpack with `--watch` in parallel?

Reply:
Also, because `migrate` and `collectstatic` will always run, this seems like it'd make dev pretty slow to start and stop?